tests/test_main.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from subprocess import CompletedProcess  # noqa: S404
+from typing import TYPE_CHECKING
+
+from pipdeptree.__main__ import main
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    import pytest
+    from pytest_mock import MockFixture
+
+
+def test_main_log_resolved(tmp_path: Path, mocker: MockFixture, capsys: pytest.CaptureFixture[str]) -> None:
```

Comment:
We actually have a `test_pipdeptree.py` file that we can throw this test in so let's add it there (the name of this file is a relic from older versions where all the code existed in a single module called `pipdeptree`). Let's add it right under the test `test_main()`.